src/main/assemblies/plugin.xml

```diff
@@ -5,6 +5,13 @@
         <format>zip</format>
     </formats>
     <includeBaseDirectory>false</includeBaseDirectory>
+    <files>
```

Comment:
The Plugin architecture has changed, now one specifies stuff in an external properties file.